src/id/doujindesu/src/eu/kanade/tachiyomi/extension/id/doujindesu/DoujinDesu.kt

```diff
@@ -368,29 +377,60 @@ class DoujinDesu : ParsedHttpSource(), ConfigurableSource {
 
     override fun mangaDetailsParse(document: Document): SManga {
         val infoElement = document.select("section.metadata").first()!!
+        val AuthorName = when {
+            document.select("section.metadata > table:nth-child(2) > tbody > tr.pages > td:contains(Author) + td:nth-child(2) > a")
```

Comment:
This is very fragile. Consider something like `.metadata table tr td:contains(Author) ~ td`

Reply:
I Will seize the selector